tests/Integration/TestCase.php

```diff
@@ -92,27 +92,4 @@ protected function loadTestDataConfig() {
 
 		$this->config = $this->getTestData( dirname( $filename ), basename( $filename, '.php' ) );
 	}
-
```

Comment:
These methods are used in quite a number of tests, why move them outside of the base testCase to a trait?